tests/integ/test_large_query_breakdown.py

```diff
@@ -426,19 +447,6 @@ def test_sort(session):
     assert len(plan_queries["post_actions"]) == 1
     assert plan_queries["post_actions"][0].startswith("DROP  TABLE  If  EXISTS")
 
-    # when sort is applied on the final dataframe, it should not be broken down
-    union_df = df1.union_all(df2)
-    final_df = union_df.with_column("A", col("A") + lit(1)).order_by("A")
-
```

Comment:
The older version of this test was ensuring that when there is a sort node within the pipeline, we break it down, and if the sort is at root node, we do not break it down.

With cutting select statement nodes being allowed, we will cut at a select statement even when only pipeline breaker(sort) is at root.

Updated test now only ensures that the final result with and without large query breakdown are same as when optimization is disabled.